### PR TITLE
Luakit support

### PR DESCRIPTION
**Pull request by EvanKrall**
This generalizes the python to be able to launch luakit as well as uzbl. Similar classes could be made for other browsers if desired.

This does not do basic auth yet, though luakit will parse username/password out of the url.

### CODE REVIEW

File: files/uzblmonitor [hunk 1]
```diff
@@ -52,6 +44,16 @@ class Browser(object):
             self.sock.close()
             self.sock = None
 
+    def read_config(self):
+        config = ConfigParser.RawConfigParser()
+        config.read(CONFIG_FILE)
+        return config
+
+    def reload_config(self):
```
Comment (patricklucas) [inline files/uzblmonitor:54]: We could just get rid of supporting reload and just let upstart restart it. That's how we do it already; might as well make it official.

File: manifests/init.pp [hunk 2]
```diff
@@ -78,6 +82,10 @@
     mode   => '0755',
     source => 'puppet:///modules/uzblmonitor/uzblmonitor',
   } ->
+  file { '/etc/default/uzblmonitor':
+    content => "BROWSER=${quoted_browser}",
+    notify => Service['uzblmonitor'],
```
Comment (patricklucas) [inline manifests/init.pp:89]: align arrows

File: files/uzblmonitor (hunk 1 above)
Comment (EvanKrall) [inline files/uzblmonitor:54]: 👍

**Comment by ziolino** [comment]
hallo,
I'm not familiar with github,
so sorry if I put this in the wrong place.

I'm running luakit on archbang,
and since almost two months I can't see flash videos.
~/.config/luakit/globals.lua is:

["youtube.com"] = {
        enable_scripts = true,
        enable_plugins = true,
    },

Even jumanji is not playing.
Firefox instead works with flash.

youtube site is configured to use html5 when possible.
Is it maybe html5 not working?

Any help appreciated,
thanks!

**Comment by EvanKrall** [comment]
@ziolino I don't know the answer to your question, sorry. I think you'd get more help if you filed an issue at https://github.com/mason-larobina/luakit instead

**Comment by ziolino** [comment]
@EvanKrall Thanks, I will.

**Pull request merged**